Readme/newcomer infos (#354)
developit/microbundle: README.md

- Produces tiny, optimized code for all inputs
- Supports multiple entry modules _(`cli.js` + `index.js`, etc)_
- Creates multiple output formats for each entry _(<abbr title="CommonJS (node)">CJS</abbr>, <abbr title="Universal Module Definition">UMD</abbr> & <abbr title="ECMAScript Modules">ESM</abbr>)_

> - 0 configuration Typecript support

Reviewer: ```suggestion
- 0 configuration TypeScript support
```

Reviewer: You could also say "No" instead of "0" but that's just choice of words.

- Built-in Terser compression & gzipped bundle size tracking

## 🔧 Installation

### Download

`npm i -D microbundle`

### Set up your `package.json`

```js
{
  "source": "src/foo.js",         // Your source file (same as 1st arg to microbundle)

>   "main": "dist/foo.js",          // Desire output path

Reviewer: ```suggestion
  "main": "dist/foo.js",          // Desired output path
```

>   "main": "dist/foo.js",          // Desired output path

Reviewer 2: ```suggestion
  "main": "dist/foo.js",        // output path for CommonJS/Node
  "module": "dist/foo.mjs",     // output path for JS Modules
  "unpkg": "dist/foo.umd.js",   // optional, for unpkg.com
```

  "scripts": {

>     "build": "microbundle",       // uses the "source" and "main" key in the file as input and output paths by default

Reviewer 2: ```suggestion
    "build": "microbundle",       // uses "source" and "main" as input and output paths by default
```

### `microbundle` / `microbundle build`

> Unless overridden via command line, microbundle uses the `source` property in your `package.json` to locate the input file, and the `main` property for the output.

Reviewer: ```suggestion
Unless overridden via the command line, microbundle uses the `source` property in your `package.json` to locate the input file, and the `main` property for the output.
```

For UMD builds, microbundle will use a snake case version of the `name` field in your `package.json` as export name. This can be overridden either by providing an `amdName` key in your `package.json` or via the `--name` flag in the cli.

### `microbundle watch`

> Just like `microbundle build`, but watches your source files and rebuilds on any change.

Reviewer: ```suggestion
Acts just like `microbundle build`, but watches your source files and rebuilds on any change.
```

> ### Using with typescript

Reviewer: ```suggestion
### Using with TypeScript
```

> Just point to a `.ts` file as input using either the cli or the `source` key in your `package.json` and you’re done.

Reviewer: ```suggestion
Just point the input to a `.ts` file through either the cli or the `source` key in your `package.json` and you’re done.
```

### Specifying builds in `package.json`

You can specify output builds in a `package.json` as follows:

```
"main": "dist/foo.js",          // CJS bundle
"umd:main": "dist/foo.umd.js",  // UMD bundle
"module": "dist/foo.mjs",       // ES Modules bundle
"source": "src/foo.js",         // custom entry module (same as 1st arg to microbundle)

> "types": "dist/foo.d.ts",       // Typescript typings

Reviewer: ```suggestion
"types": "dist/foo.d.ts",       // TypeScript typings
```
